src/components/Container/Container.scss

```diff
@@ -0,0 +1,25 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  max-width: $breakpoint-laptop-large;
+  padding: 5px;
+  margin: 0 16px;
+  text-align: center;
+}
+// TODO include an escape before this loop
```

Comment:
This can't be merged until that escape is here this will throw errors for everyone else, see mike's comment about escaping https://sf-alumni.slack.com/archives/G017D2053T8/p1597105852038300

Reply:
@mattlaguardia I've included the escape, and committed it.